crates/nsed-orchestrator/Makefile

```
@@ -0,0 +1,155 @@
+
+.PHONY: check-orch test-orch run run-sim dev dev-sim schema
+.PHONY: nats-reset
+.PHONY: auth-token auth-token-new auth-check auth-setup
+
+# When included from root, ROOT_DIR is already set to ".".
+# When running standalone, resolve via git.
+ROOT_DIR ?= $(shell git rev-parse --show-toplevel 2>/dev/null || echo ../..)
+
+# env-check fallback for standalone use (root overrides this via _ENV_CHECK_DEFINED).
+ifndef _ENV_CHECK_DEFINED
+.PHONY: env-check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)